src/source.js

```diff
@@ -0,0 +1,55 @@
+// import '@fortawesome/fontawesome-free/js/fontawesome';
```

Comment:
- [ ] Let's rename this file to `index.js` to meet this requirement

![](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/src_requirement.png)

Reply:
Anyway, I think naming is just a convention